mailchimp.api.php

```diff
@@ -56,3 +56,43 @@ function hook_mailchimp_subscribe_user($list_id, $email, $merge_vars) {
 function hook_mailchimp_unsubscribe_user($list_id, $email) {
 
 }
+
```

Comment:
Would it make sense to put these in `modules/mailchimp_automations/mailchimp_automations.api.php`?